packages/@webex/plugin-meetings/test/unit/spec/locus-info/index.js

```diff
@@ -739,46 +739,44 @@ describe('plugin-meetings', () => {
 
     describe('#updateSelf', () => {
       it('should trigger SELF_MEETING_BRB_CHANGED when brb state changed', () => {
-        locusInfo.self = undefined
+        locusInfo.self = undefined;
 
         const assertBrb = (enabled) => {
           const selfWithBrbChanged = cloneDeep(self);
-          selfWithBrbChanged.controls.brb = enabled
+          selfWithBrbChanged.controls.brb = enabled;
 
           locusInfo.emitScoped = sinon.stub();
-          locusInfo.updateSelf(selfWithBrbChanged, [])
+          locusInfo.updateSelf(selfWithBrbChanged, []);
 
           assert.calledWith(
             locusInfo.emitScoped,
             { file: 'locus-info', function: 'updateSelf' },
             LOCUSINFO.EVENTS.SELF_MEETING_BRB_CHANGED,
             { brb: enabled }
           )
-        }
+        };
 
-        assertBrb(true)
-        assertBrb(false)
+        assertBrb(true);
+        assertBrb(false);
       })
 
-      it('should not trigger SELF_MEETING_BRB_CHANGED when brb state changed', () => {
+      it('should not trigger SELF_MEETING_BRB_CHANGED when brb state did not change', () => {
         const assertBrb = (enabled) => {
-          const selfWithBrbChanged = cloneDeep(self);
-          selfWithBrbChanged.controls.brb = enabled
 
-          locusInfo.self = selfWithBrbChanged
+          locusInfo.self = undefined;
           locusInfo.emitScoped = sinon.stub();
-          locusInfo.updateSelf(selfWithBrbChanged, [])
+          locusInfo.updateSelf(undefined, []);
```

Comment:
Is this a valid case? Will we ever actually call `locusInfo.updateSelf` with `undefined` when `locusInfo.self` is undefined?

Is it also possible for `locusInfo.self` to be defined and the `locusInfo.updateSelf` to be called with the `brb` state unchanged? Should we test that also?

Reply:
I agree with the idea to test that brb is not changed. it's a good one/ for `undefined` @edvujic, could you please check if this flow is actually possible, and if not, then it would be good to update it to Bryce's suggestion with brb.